spot-client/src/spot-tv/native-functions/bt-beacon/middleware.js

```diff
@@ -0,0 +1,27 @@
+import { SPOT_TV_SET_JOIN_CODE } from 'common/app-state/spot-tv/action-types';
```

Comment:
Why did you choose to import from the action-types directory instead of getting from common/app-state?

Reply:
Dunno. I guess I thought it's logical :)

I'll fix this, thanks

Reply:
Oh, I guess `SPOT_TV_SET_JOIN_CODE` doesn't get exported. I never had middleware planned for jitsi-meet-spot and never thought something outside of app-state would need the action type. So...undo? Or maybe think of how you would want it?

Reply:
Nah, added the export. But I'm hesitating:
Exporting too much on the top level leads easily to circular deps. But it makes the code tidy. What do you think?